ui/src/app/settings/views/Scripts/ScriptsList.js

```diff
@@ -155,6 +156,9 @@ const ScriptsList = ({ type = "commissioning" }) => {
   const userScripts = useSelector(state =>
     scriptSelectors.search(state, searchText, type)
   );
+
+  useWindowTitle(`${type} scripts`);
```

Comment:
Might need to capitalize the type here because it's showing as e.g "commissioning scripts"

Reply:
Ah yeah, good spot, thanks.